libc/utils/docgen/ctype.json

```diff
@@ -1,7 +1,46 @@
-{
-  "functions": {
-    "isalnum": null,
-    "isalpha": null,
-    "isblank": null
-  }
+{	
+	"functions": {
+		"isalnum": {
+			"defined": "7.4.1.1"
+		},
```

Comment:
the indentation is still funky here.  Are you using tabs by accident rather than spaces?  Your editor should be configurable; I even go so far in my editor as to make tabs visible characters.

Worst case, you could paste this into some online formatter, format it to use 2 spaces for indentation consistently, then paste it back.

But this will continue to be an issue when you contributor to other projects, so I do recommend sorting out your editor as a proper long term fix.

Luckily, this isn't an issue for parsing the json to generate the tables, but I'd like to keep the indentation minimal and consistent.

Reply:
current version is consistent, but is still using tabs rather than 2 spaces

Reply:
fixed